deploy/eck-elasticsearch/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+
+1. Check Elasticsearch cluster status
+  $ kubectl get es --namespace={{ .Release.Namespace }} -L common.k8s.elastic.co/type=elasticsearch
+
+2. Check Elasticsearch pod status
+  $ kubectl get pods --namespace={{ .Release.Namespace }} -L common.k8s.elastic.co/type=elasticsearch
+
+3. Expose Elasticsearch service
+  $ kubectl port-forward svc/{{ .Values.name }}-es-http 9200
```

Comment:
`.Values.name` does not exist?  Also the http service naming convention is something like `{{ include "elasticsearch.fullname }}-es-http` I am not sure how important this instruction is though in practice maybe we can leave 3. off altogether?